apiserver/facades/client/spaces/spaces.go

```diff
@@ -280,7 +279,22 @@ func (api *API) ShowSpace(ctx stdcontext.Context, entities params.Entities) (par
 
 // ReloadSpaces refreshes spaces from substrate
 func (api *API) ReloadSpaces(ctx stdcontext.Context) error {
-	return api.reloadSpacesAPI.ReloadSpaces(ctx)
+	err := api.auth.HasPermission(permission.AdminAccess, api.backing.ModelTag())
+	if err != nil {
+		return errors.Trace(err)
+	}
+	if err := api.check.ChangeAllowed(ctx); err != nil {
+		return errors.Trace(err)
+	}
+	modelConfig, err := api.backing.ModelConfig(ctx)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	fanConfig, err := modelConfig.FanConfig()
+	if err != nil {
+		return errors.Trace(err)
+	}
```

Comment:
You can access the model config from inside the reload spaces, so you don't need to pass in the FanConfig.